core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -910,6 +910,7 @@ class KafkaApis(val requestChannel: RequestChannel,
         } else if (allowAutoTopicCreation && config.autoCreateTopicsEnable) {
           createTopic(topic, config.numPartitions, config.defaultReplicationFactor)
         } else {
+          warn(s"The topic '$topic' does not exist and auto topic creation is disabled.")
```

Comment:
Since auto-topic creation is disabled by default, not clear that we should be logging in the broker for these cases.

Reply:
@ijuma, thanks for the feedback. Do you think the JIRA is not valid then, or should we reduce the severity of this log (maybe to debug)?